code/modules/spells/spell_types/shapeshift.dm

```diff
@@ -14,10 +14,10 @@
 	var/shapeshift_type
 	var/list/current_shapes = list()
 	var/list/current_casters = list()
-	var/list/possible_shapes = list(/mob/living/simple_animal/mouse,\
-		/mob/living/simple_animal/pet/dog/corgi,\
-		/mob/living/simple_animal/hostile/carp/ranged/chaos,\
-		/mob/living/simple_animal/bot/ed209,\
+	var/list/possible_shapes = list(/mob/living/simple_animal/mouse,
+		/mob/living/simple_animal/pet/dog/corgi,
+		/mob/living/simple_animal/hostile/carp/ranged/chaos,
+//		/mob/living/simple_animal/bot/ed209,
```

Comment:
Keep in mind ED's are susceptible to ions. As well as lasers. Tasers do not go through windows, which lasers do. ED's have far less durability than a carbonmob on top of that.

Also keep in mind that wizard is balanced around Single Person Vs Entire Station and not Bullshit Ragin. Ragin mages was always meant to be an adminbus gamemode and was never meant to be in regular rotation. The shittery we see daily is because we not only have it in rotation, but also have it weighted very high for some reason.